saleor/payment/__init__.py

```diff
@@ -108,7 +108,7 @@ class ChargeStatus:
 GATEWAYS_ENUM = Enum(
     'GatewaysEnum',
     {key.upper(): key.lower()
-     for key in settings.PAYMENT_GATEWAYS})
+     for key in settings.CHECKOUT_PAYMENT_GATEWAYS})
```

Comment:
That's already handled at the resolver level, see `Checkout.availablePaymentGateways` field.

Reply:
I mean the ```availablePaymentGateways``` will only return the available gateway indeed, but from the docs, you can still find other payment gateways which is from ```GATEWAYS_ENUM```. Since these gateways are not enabled, should we expose it in the API even only in the docs?

Reply:
OK, I think we can change it that way and only return those that are enabled in the enum as well.